openapi.yml

```diff
@@ -7466,23 +7466,69 @@ paths:
         GRANT CREATE USER ON ACCOUNT TO ROLE "KEBOOLA_STORAGE" WITH GRANT
         OPTION;
 
-        CREATE WAREHOUSE "YOUR_WAREHOUSE" WITH WAREHOUSE_SIZE = 'XSMALL'
-        WAREHOUSE_TYPE = 'STANDARD' AUTO_SUSPEND = 300 AUTO_RESUME = TRUE;
+        CREATE WAREHOUSE "KEBOOLA" WITH WAREHOUSE_SIZE = 'XSMALL' WAREHOUSE_TYPE
+        = 'STANDARD' AUTO_SUSPEND = 60 AUTO_RESUME = TRUE;
 
-        GRANT USAGE ON WAREHOUSE "YOUR_WAREHOUSE" TO ROLE "KEBOOLA_STORAGE" WITH
-        GRANT OPTION;
+        GRANT USAGE ON WAREHOUSE "KEBOOLA" TO ROLE "KEBOOLA_STORAGE" WITH GRANT
+        OPTION;
 
 
         CREATE USER "KEBOOLA_STORAGE"
+            PASSWORD = "YOUR_PASSWORD"
+            DEFAULT_ROLE = "KEBOOLA_STORAGE";
 
-        PASSWORD = "YOUR_PASSWORD"
+        GRANT ROLE "KEBOOLA_STORAGE" TO USER "KEBOOLA_STORAGE";
```

Comment:
preco sa tu menia tieto veci ked je to PR ohladne project feature management? A imho tieto veci tu asi mozu zstat tak ako su...

Reply:
Tohel je jen přegenerované `openapi.yml` pomocí

```
$ cat apiary.apib | grep -v "X-KBC-ManageApiToken:" | apib2swagger -o openapi.yml -y --open-api-3 --info-title="Manage API" 
$ php AdjustOpenApi.php
```

protože na to někdo předtím evidentně zapomněl.

Reply:
hodil jsem to do samostatného MR: https://github.com/keboola/kbc-manage-api-php-client/pull/258

Reply:
ale proč mi tam generátor udělal ty změny na konci řádků v mezerách, to nevím …